### ISSUE TITLE
support for --starting-state-exit-code

### ISSUE THREAD

**Issue by prupert** [vint21h/nagios-check-supervisord#9]
If a program is "starting", currently the script will exit with a WARNING exit code. In most cases it may be desirable to accept a "starting" program as normal behaviour. Example:

```
WARNING: something curiously with 'example-api-client_01': (STARTING), 'example-api-client_00': OK, 'example-api-default_00': OK, 'example-api-default_01': OK, 'example-api-default_02': OK, 'example-api-high_00': OK, 'example-api-high_01': OK
```

Would it be possible to add an option for `--starting-state-exit-code`, in a similar fashion to `--stopped-state-exit-code`?

**Comment by prupert** [comment]
Pull request submitted for review at https://github.com/vint21h/nagios-check-supervisord/pull/10

**Comment by vint21h** [comment]
Accepted, merged, and available in the 2.1.0 release.
Thanks for your contribution.

**Comment by prupert** [comment]
Great. I think we should also update the readme. ~~Can work on a PR when I have time later today.~~ See https://github.com/vint21h/nagios-check-supervisord/pull/11
